ext/strings_test.go

```diff
@@ -29,13 +29,27 @@ var stringTests = []struct {
 }{
 	// CharAt test.
 	{expr: `'tacocat'.charAt(3) == 'o'`},
-	// Indext of search string tests.
-	{expr: `'tacocat'.indexOf('a') == 1`},
-	{expr: `'tacocat'.indexOf('a', 3) == 5`},
+	{expr: `'tacocat'.charAt(7) == ''`},
+	{expr: `'©αT'.charAt(0) == '©' && '©αT'.charAt(1) == 'α' && '©αT'.charAt(2) == 'T'`},
+	// Index of search string tests.
+	{expr: `'tacocat'.indexOf('') == 0`},
+	{expr: `'tacocat'.indexOf('ac') == 1`},
 	{expr: `'tacocat'.indexOf('none') == -1`},
-	{expr: `'tacocat'.lastIndexOf('a') == 5`},
-	{expr: `'tacocat'.lastIndexOf('a', 3) == 1`},
+	{expr: `'tacocat'.indexOf('', 3) == 3`},
+	{expr: `'tacocat'.indexOf('a', 3) == 5`},
+	{expr: `'tacocat'.indexOf('at', 3) == 5`},
+	{expr: `'ta©o©αT'.indexOf('©') == 2`},
+	{expr: `'ta©o©αT'.indexOf('©', 3) == 4`},
+	{expr: `'ta©o©αT'.indexOf('©αT', 3) == 4`},
+	{expr: `'ta©o©αT'.indexOf('©α', 5) == -1`},
+	{expr: `'tacocat'.lastIndexOf('') == 7`},
+	{expr: `'tacocat'.lastIndexOf('at') == 5`},
 	{expr: `'tacocat'.lastIndexOf('none') == -1`},
+	{expr: `'tacocat'.lastIndexOf('', 3) == 3`},
+	{expr: `'tacocat'.lastIndexOf('a', 3) == 1`},
+	{expr: `'ta©o©αT'.lastIndexOf('©') == 4`},
+	{expr: `'ta©o©αT'.lastIndexOf('©', 3) == 2`},
+	{expr: `'ta©o©αT'.lastIndexOf('©α', 4) == 4`},
 	// Replace tests
```

Comment:
Add unicode tests for replace and split.  Yes, you're just calling predefined string library functions, so you don't need to go crazy, but just have one unicode character returned to make sure.

Reply:
Done.